README.md

```diff
@@ -1,89 +1,162 @@
-and the platform on which the operation will be solved. The first analysis
-studies the accuracy of the solution while the second one compares the
-performances of the different implementations to select the best one.
-
-Nowadays, all the numerical computations are based on a set of standard
-libraries on which the most common operations are implemented. These libraries
-are different for dense matrices (BLAS, LAPACK, ScaLAPACK, ...) and for sparse
-matrices (SparseBLAS, ...). Moreover, there are  vendor implementations which
-are adjusted to the platform features:
-  - For multicores: ACML (AMD), ATLAS, Intel-MKL, OpenBLAS, ...
-  - For GPUs: cuBLAS (Nvidia), clBLAS, CLBlast, MAGMA, ...
-
-But, in any case, BLAS is always the lowest level in the hierarchy
-of numerical libraries, such that
+BLAS is always the lowest level in the hierarchy of numerical libraries, such that
 a good BLAS implementation improves the performances of all the other
-libraries.  The development of numerical libraries on SYCL is one of the most
-important objectives, because it will improve the performance of other SYCL
-applications. Obviously, it makes sense SYCL-BLAS was the first step in this
-task.
-
-On GPUs, the data communication to/from the device and the grain of the kernels
-play an important rule on the performances of the developments. On one
-hand, to reduce the communication cost, the most of the data should be mapped
-on the device, even the scalars. On the other hand, growing the size of the
-kernels allows the CPU to complete other tasks while the GPU is computing or to
-enter an energy-efficient C-state, reducing the energy consumption.
-
-To enlarge the grain of the kernels is a complex task, in which many aspects
-should be considered as the dependency between kernels, the grid topology, the
-grid sizes, etc. This complexity justifies that, usually, the fused kernels are
-manually written. An alternative to simplify this task could be to build a
-expression tree on which all the single operation which are required to solve a
-problem appears. This structure could be analysed by the compiler to decide how
-to merge the different kernel and the best grid topology to execute the fused
-kernel.  The use of expression trees is one of most important features of
-SYCL-BLAS.
-
-## Basic Concepts
+libraries.
+
+The SYCL-BLAS project is used by the [SYCL-DNN project](https://github.com/codeplaysoftware/SYCL-DNN) 
+project as a supporting library to perform some operations. SYCL-DNN is a library 
+implementing neural network operations.
+
+SYCL-BLAS has been developed in collaboration with the 
+*High Performance Computing & Architectures (HPCA) group*
+from the Universitat Jaume I [UJI](http://www.hpca.uji.es/).
+
+The project is maintained by [Codeplay Software](https://developer.codeplay.com).
+
+## Project Requirements
+
+SYCL-BLAS is designed to work with any SYCL 1.2.1 implementation.
+* We do not use any OpenCL interoperability so the code is pure C++.
```

Comment:
I think the GEMM backend for IMGDNN  uses OpenCL interoperability mode.

Reply:
I've removed this point since it is not correct any longer.